public/v4/apps/silex-platform.yml

```diff
@@ -0,0 +1,69 @@
+captainVersion: 4
+services:
+    '$$cap_appname':
+        image: silexlabs/silex-platform:$$cap_version
+        environment:
+            SILEX_URL: https://$$cap_appname.$$cap_root_domain
+            STORAGE_CONNECTORS: $$cap_storage_connector
+            HOSTING_CONNECTORS: $$cap_hosting_connector
+            GITLAB_DOMAIN: https://gitlab.com
+            GITLAB_CLIENT_ID: $$cap_gitlab_client_id
+            GITLAB_CLIENT_SECRET: $$cap_gitlab_client_secret
+            GITLAB2_DOMAIN: $$cap_gitlab2_domain
+            GITLAB2_CLIENT_ID: $$cap_gitlab2_client_id
+            GITLAB2_CLIENT_SECRET: $$cap_gitlab2_client_secret
+            GITLAB_SCOPE: 'api read_user'
+            GITLAB_TIMEOUT: 5000
+        volumes:
+            # - '$$cap_appname-data:/srv/data'
+            # - '$$cap_appname-cfg:/srv/cfg'
+        caproverExtra:
+            containerHttpPort: '6805'
+caproverOneClickApp:
+    variables:
+        - id: '$$cap_version'
+          label: silex-platform Version
+          defaultValue: 'v1.0.162'
+          description: Check out their Docker page for the valid tags https://hub.docker.com/r/silexlabs/silex-platform/tags
+          validRegex: "/^([^\\s^\\/])+$/"
+        - id: '$$cap_tz'
```

Comment:
I don't see this one used anywhere else, it's probably useless?

Reply:
done